src/promptflow/tests/sdk_cli_test/e2etests/test_cli.py

```diff
@@ -2160,6 +2160,107 @@ def get_successful_lines(output_path):
+        assert "Completed" in out
+        original_run = local_client.runs.get(name=run_id)
+
+        new_run_id = str(uuid.uuid4())
+        display_name = "test"
+        description = "new description"
+        run_pf_command(
+            "run",
+            "create",
+            "--resume-from",
+            run_id,
+            "--name",
+            new_run_id,
+            "--set",
+            f"display_name={display_name}",
+            f"description={description}",
+            "tags.A=A",
+            "tags.B=B",
+        )
+        resume_run = local_client.runs.get(name=new_run_id)
+        assert resume_run.name == new_run_id
+        assert resume_run.display_name == display_name
+        assert resume_run.description == description
+        assert resume_run.tags == {"A": "A", "B": "B"}
+        assert resume_run._resume_from == run_id
+        assert (
+            original_run.properties["system_metrics"]["total_tokens"]
+            <= resume_run.properties["system_metrics"]["total_tokens"]
+        )
+
+    def test_flow_run_resume_from_image_aggregation(self, capfd, local_client) -> None:
+        run_id = str(uuid.uuid4())
+        # fetch std out
+        run_pf_command(
+            "run",
+            "create",
+            "--flow",
+            f"{FLOWS_DIR}/eval_flow_with_image_resume_random_fail",
+            "--data",
+            f"{FLOWS_DIR}/eval_flow_with_image_resume_random_fail/data.jsonl",
+            "--column-mapping",
+            "input_image='${data.input_image}'",
+            "--name",
+            run_id,
+        )
+        out, _ = capfd.readouterr()
+        assert "Completed" in out
+        original_run = local_client.runs.get(name=run_id)
+        output_path = os.path.join(original_run.properties["output_path"], "flow_outputs", "output.jsonl")
+        with open(output_path, "r") as file:
+            original_output = [json.loads(line) for line in file]
+
+        new_run_id = str(uuid.uuid4())
+        display_name = "test"
+        description = "new description"
+        run_pf_command(
+            "run",
+            "create",
+            "--resume-from",
+            run_id,
+            "--name",
+            new_run_id,
+            "--set",
+            f"display_name={display_name}",
+            f"description={description}",
+            "tags.A=A",
+            "tags.B=B",
+        )
+        resume_run = local_client.runs.get(name=new_run_id)
+        output_path = os.path.join(resume_run.properties["output_path"], "flow_outputs", "output.jsonl")
+
+        with open(output_path, "r") as file:
+            resume_output = [json.loads(line) for line in file]
+
+        # assert original_output in resume_output
+        original_output_line_numbers = {line["line_number"] for line in original_output}
+        resume_output_line_numbers = {line["line_number"] for line in resume_output}
+        assert original_output_line_numbers.issubset(resume_output_line_numbers)
```

Comment:
Please use `log_metric` to save input image count of aggregation node. That will result in a user metric.
Then you can assert that the metric value of resume run is larger than the original run.

Reply:
Added metric logger in sdk_cli_test and checked pf.runs.get_metrics in sdk_cli_azure_test.